Update enhanced-filtering-for-connectors.md (#1982)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mail-flow-best-practices/use-connectors-to-configure-mail-flow/enhanced-filtering-for-connectors.md

       - This option is only affective on the actual email address that you specify. For example, if a user has five email addresses associated with their mailbox (also known as _proxy addresses_), you'll need to specify all five of their email addresses here. Otherwise, messages that are sent to the four other email addresses will go through normal filtering.

>         - For hybrid environment which the inbound mail flow is through Exchange On-Premises, you'll need to specify the targetAddress of the MailUser object. E.g.: michelle@contoso.mail.onmicrosoft.com

Reviewer: ```suggestion
        - For a hybrid environment in which the inbound mail flow is through on-premises Exchange, you'll need to specify the targetAddress of the MailUser object, for example, _michelle@contoso.mail.onmicrosoft.com_.
```

```

> [!IMPORTANT]

> > Entering the IP addresses of Office 365 is not supported. Do not use this feature to compensate for issues introduced by unsupported email routing paths. Use caution and limit the IP ranges to only the mail systems that will handle your own organization's messages prior to Office 365. Entering private IP address is not supported as well. Enhanced Filtering automatically detect and skip private IP addresses.

Reviewer: ```suggestion
> Entering the IP addresses of Office 365 is not supported. Do not use this feature to compensate for issues introduced by unsupported email routing paths. Use caution and limit the IP ranges to only the mail systems that will handle your own organization's messages prior to Office 365. Entering a private IP address is also not supported. Enhanced Filtering automatically detects and skips private IP addresses.
```
